Packs/F5Silverline/Integrations/F5Silverline/F5Silverline.py

```diff
@@ -194,20 +197,58 @@ def delete_ip_objects_command(client: Client, args: dict[str, Any]):
                                       'list_target': list_target}, resp_type='content')
             human_readable += f"IP object with ID: {object_id} deleted successfully from the {list_type} list. \n"
         return CommandResults(readable_output=human_readable)
+    return None
 
 
 def get_object_id_by_ip(client, list_type, object_ip):
-    url_suffix = f'{list_type}/ip_objects'
-    response = client.request_ip_objects(body={}, method='GET', url_suffix=url_suffix, params={})
-    all_objects = response.get('data')
+    """Get object ID by its IP attribute.
+    This function paginates over a list of objects and extracts all of their ID attributes as preparation for deletion.
+
+    Note: this function, used by the 'delete_ip_objects_command' command is utilizing pagination.
+    This is because the GET request is limited to return up to 1,000 objects per requests,
+    which may result in 404 error if the user has overall more objects than that.
+    while the 'get_ip_objects_list_command' command already utilizes pagination,
+    this is a different use-case since there is no use user input regarding pagination in
+    'f5-silverline-ip-object-delete command (there is no need).
+
+    Args:
+        client (Client): Client object
+        list_type (list): Type of list to query
+        object_ip (_type_): Object IP to search for deletion.
+
+    Raises:
+        DemistoException: Raised in the case an object with the given IP address was not found.
+
+    Returns:
+        list: A list of found ID objects.
+    """
+
+    page = 1
+    url_suffix = f"{list_type}/ip_objects"
+    response = client.request_ip_objects(body={}, method="GET", url_suffix=url_suffix, params={})
+    all_objects: list = response.get("data")
+    try:
+        while next_page := response.get("links", {}).get("links", {}).get("next", {}):
```

Comment:
Why not using demisto.get?

Reply:
The response structure have "link" key within a "link" key, and i'm pretty sure demisto.get wont be able to handle that, thats why I did it explicitly.